lib/datadog/statsd.rb

```diff
@@ -78,7 +79,7 @@ def tags
     def initialize(
       host = nil,
       port = nil,
-      socket_path: nil,
+      socket_path: ENV['DD_DOGSTATSD_SOCKET'],
```

Comment:
Now that `ConnectionCfg` reads the environment variables directly, I'm not sure it makes sense to have this change here.

Having it here means that if a customer deploys to an environment that had a leftover `DD_DOGSTATSD_SOCKET`, they will start seeing the `ArgumentError, "Do not set both host/port and socket_path"` from `ConnectionCfg` which seems a bit confusing because they may not realize what they're doing wrong until they look at the gem code.

Reply:
Indeed -- I intended to remove this and forgot.  Thanks for spotting it!